plugins/board-resources/src/components/editor/CardActions.svelte

```diff
@@ -14,30 +14,81 @@
 // limitations under the License.
 -->
 <script lang="ts">
-  import type { Card } from '@anticrm/board'
+  import board from '@anticrm/board'
+  import type { Card, CardAction } from '@anticrm/board'
+  import { IntlString, getResource } from '@anticrm/platform'
+  import { getClient } from '@anticrm/presentation'
   import { Button, Label } from '@anticrm/ui'
-  import { getEditorCardActionGroups } from '../../utils/CardActionUtils'
+
+  import plugin from '../../plugin'
+  import { cardActionSorter, getCardActions } from '../../utils/CardActionUtils'
 
   export let value: Card
+  const client = getClient()
+
+  let addToCardActions: CardAction[] = []
+  let automationActions: CardAction[] = []
+  let actions: CardAction[] = []
+
+  let actionGroups: { label: IntlString; actions: CardAction[] }[] = []
+
+  getCardActions(client).then((result) => {
+    result.forEach(async (action) => {
```

Comment:
It will not work, because .forEach doesn't await inner func. You should use `for of` or any another loop and await all promises. Currently sort may work incorrect